.gitignore

```diff
@@ -25,4 +25,5 @@ onigumo-*.tar
 # Temporary files, for example, from tests.
 /tmp/
 
-onigumo
+# Ignore onigumo escript file
```

Comment:
This would theoratically belong into a different pull request. 😃 The fact that the line doesn’t have a comment is not related to the actual bug in the definitilon. Sorry for nitpicking.